### PR TITLE
Fixed/Cleaned up Readme and Added/Fixed Facebook links in footer

### PR DESCRIPTION
**Pull request by devinmatte**
- Reformatted README with proper markdown
- Added Public Facebook link to footer
- Fixed Facebook group link to proper location
- Added .idea to the .gitignore
- Removed a duplicate *.tmp from gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -163,3 +162,4 @@ pip-log.txt
 
 # Mac crap
 .DS_Store
+.idea
```
Comment (liam-middlebrook) [inline .gitignore]: What is a .idea? Why should we be ignoring it if it's not a generated filetype in the repo?

Comment (liam-middlebrook) [inline .gitignore]: Also please fix so the file has UNIX style file endings

File: resources/templates/_footer.html [hunk 2]
```diff
@@ -56,10 +56,16 @@ <h3 style="color:#FFF;">Public Resources</h3>
                             <a href="http://schedule.csh.rit.edu/" class="sitemap-link">CSH RIT Schedule Maker</a>
                         </li>
                         <li>
-                            <a href="https://github.com/ComputerScienceHouse" class="sitemap-link"><i class="fa fa-github"></i> CSH on GitHub</a>
+                            <a href="https://github.com/ComputerScienceHouse" class="sitemap-link"><i
+                                    class="fa fa-github"></i> CSH on GitHub</a>
                         </li>
                         <li>
-                            <a href="https://twitter.com/CSH_History" class="sitemap-link"><i class="fa fa-twitter"></i> @CSH_History</a>
+                            <a href="https://www.facebook.com/RITCSH/" class="sitemap-link"><i
```
Comment (liam-middlebrook) [inline resources/templates/_footer.html]: Could you not have the line break at 80chars here? IMO it makes reading the HTML source more difficult.

File: resources/templates/_footer.html [hunk 3]
```diff
@@ -72,15 +78,18 @@ <h3 style="color:#FFF;">Member Resources</h3>
...
                         <li>
-                            <a href="https://constitution.csh.rit.edu/articles.pdf" class="sitemap-link">Constitution</a>
+                            <a href="https://constitution.csh.rit.edu/articles.pdf"
+                               class="sitemap-link">Constitution</a>
                         </li>
                         <li>
                             <a href="https://constitution.csh.rit.edu/bylaws.pdf" class="sitemap-link">By-Laws</a></li>
                         <li>
-                            <a href="https://www.facebook.com/groups/ComputerScienceHouse/" class="sitemap-link"><i class="fa fa-facebook"></i> CSH on Facebook</a>
+                            <a href="https://www.facebook.com/groups/16764593277/" class="sitemap-link"><i
```
Comment (liam-middlebrook) [inline resources/templates/_footer.html]: Why the change to ID instead of name?

File: .gitignore (hunk 1 above)
Comment (devinmatte) [inline .gitignore]: .idea is the folder generated by any Jetbrains ide

File: resources/templates/_footer.html (hunk 3 above)
Comment (devinmatte) [inline resources/templates/_footer.html]: Name doesn't resolve. It doesn't exist according to Facebook

File: resources/templates/_footer.html (hunk 2 above)
Comment (devinmatte) [inline resources/templates/_footer.html]: Fixed in recent commit

File: resources/templates/_footer.html [hunk 4]
```diff
@@ -77,10 +80,10 @@ <h3 style="color:#FFF;">Member Resources</h3>
                         <li>
                             <a href="https://constitution.csh.rit.edu/bylaws.pdf" class="sitemap-link">By-Laws</a></li>
                         <li>
-                            <a href="https://www.facebook.com/groups/ComputerScienceHouse/" class="sitemap-link"><i class="fa fa-facebook"></i> CSH on Facebook</a>
+                            <a href="https://www.facebook.com/groups/16764593277/" class="sitemap-link"><i class="fa fa-facebook"></i> CSH Facebook Group</a>
```
Comment (liam-middlebrook) [inline resources/templates/_footer.html]: I don't think this should be included. I'm pretty sure this was supposed to be a link to a public page.

Comment (liam-middlebrook) [inline resources/templates/_footer.html]: As in this entire list element should be removed.

Comment (devinmatte) [inline resources/templates/_footer.html]: Hmm alright. I mean we haven't had a public page until a month ago and I know for a fact that link was there when I first visited the site a year ago. It's under member resources so I expect it was meant to be the group link

Comment (devinmatte) [inline resources/templates/_footer.html]: @brandonhudson should know the intention. Do we want the group link on the footer with the public page? Or just one?

**Comment by devinmatte** [comment]
Updated with the Removal of Private Facebook Group

**Comment by devinmatte** [comment]
Is there anything else I need to do before this gets merged or has just no one gotten around to making the merge?
